modules/exploits/unix/webapp/pfadmin_protected_alias.rb

```diff
@@ -0,0 +1,141 @@
+
+
+require 'msf/core'
+
+class MetasploitModule < Msf::Exploit::Remote
```

Comment:
If you decide to keep the library, you will need to add RSpec tests. See WordPress and Joomla...